src/frontends/pytorch/src/op_table.cpp

```diff
@@ -468,6 +468,11 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::repeat_interleave", op::translate_repeat_interleave},
         {"aten::reshape", op::translate_reshape},
         {"aten::reshape_as", op::translate_reshape_as},
+        // TO DO: enable behaviour for resolve_conj and resolve_neg complex tensors,
+        // when complex dtype will be supported
+        // for real dtypes, these operations return input tensor without changes and can be skipped
+        {"aten::resolve_conj", op::skip_node},
+        {"aten::resolve_neg", op::skip_node},
```

Comment:
Please consider to move `ComplexTypeMark` to common frontend and implement normal translators.
Create JIRA ticket for this to set in `TODO`.
Also, it is a bit strange that the model uses such operations and as if there are no complex tensors.

Reply:
as it was discussed yesterday with @slyalin I'll not do that for now

Reply:
Let us create JIRA ticket for that then.

Reply:
Let's create a ticket and do not block.